buildconfig/download_msys2_prebuilt.py

```diff
@@ -18,51 +20,66 @@ def install_pacman_package(pkg_name):
     return output.returncode != 0
 
 
-def get_packages(x86=True, x64=True):
+def get_packages(MINGW_ARCH):
     deps = [
-        'mingw-w64-{}-SDL2',
-        'mingw-w64-{}-SDL2_ttf',
-        'mingw-w64-{}-SDL2_image',
-        'mingw-w64-{}-SDL2_mixer',
-        'mingw-w64-{}-portmidi',
-        'mingw-w64-{}-libpng',
-        'mingw-w64-{}-libjpeg-turbo',
-        'mingw-w64-{}-libtiff',
-        'mingw-w64-{}-zlib',
-        'mingw-w64-{}-libwebp',
-        'mingw-w64-{}-libvorbis',
-        'mingw-w64-{}-libogg',
-        'mingw-w64-{}-flac',
-        'mingw-w64-{}-libmodplug',
-        'mingw-w64-{}-mpg123',
-        'mingw-w64-{}-opus',
-        'mingw-w64-{}-opusfile',
-        'mingw-w64-{}-freetype'
+        '{}-SDL2',
+        '{}-SDL2_ttf',
+        '{}-SDL2_image',
+        '{}-SDL2_mixer',
+        '{}-portmidi',
+        '{}-libpng',
+        '{}-libjpeg-turbo',
+        '{}-libtiff',
+        '{}-zlib',
+        '{}-libwebp',
+        '{}-libvorbis',
+        '{}-libogg',
+        '{}-flac',
+        '{}-libmodplug',
+        '{}-mpg123',
+        '{}-opus',
+        '{}-opusfile',
+        '{}-freetype'
     ]
 
-    packages = []
-    if x86:
-        packages.extend([x.format('i686') for x in deps])
-    if x64:
-        packages.extend([x.format('x86_64') for x in deps])
+    MINGW_PREFIX_DICT = {
+        "mingw64":"mingw-w64-x86_64",
+        "mingw32":"mingw-w64-i686",
+        "ucrt64":"mingw-w64-ucrt-x86_64",
+        "clang64":"mingw-w64-clang-x86_64",
+        "clang32":"mingw-w64-clang-i686",
+    }
```

Comment:
This can be saved outside the function maybe? (For better clarity and reusability in other functions)